tests/unit/s2n_tls13_state_machine_handshake_test.c

```diff
@@ -394,14 +394,42 @@ int main(int argc, char **argv)
 
         conn->handshake.handshake_type = test_handshake_type;
 
-        for (int i=0; i < sizeof(expected) / sizeof(char *); i++) {
+        for (int i = 0; i < sizeof(expected) / sizeof(char *); i++) {
             conn->handshake.message_number = i;
             EXPECT_STRING_EQUAL(expected[i], s2n_connection_get_last_message_name(conn));
         }
 
         EXPECT_SUCCESS(s2n_connection_free(conn));
     }
 
+    /* Test: TLS 1.3 message types are all properly printed for client auth */
+    {
+        uint32_t test_handshake_type = NEGOTIATED | FULL_HANDSHAKE | CLIENT_AUTH;
+        const char* expected[] = { "CLIENT_HELLO",
```

Comment:
Would the test actually have captured the issue? It seems like it now requires to update the String in `expected` and the other file. If none is updated, the test still succeeds?

Reply:
it'd be caught in the next test "Make sure not to miss out populating any message names"